custom_components/magiqtouch/climate.py

```diff
@@ -168,28 +264,58 @@ def hvac_mode(self):
         if temperature_mode:
             _LOGGER.debug(HVAC_MODE_COOL)
             return HVAC_MODE_COOL
+        heating_running = self.controller.current_state.HRunning
+        if heating_running:
+            if self.controller.current_state.HFanOnly:
+                _LOGGER.debug(HVAC_MODE_FAN_ONLY)
+                return HVAC_MODE_FAN_ONLY
```

Comment:
Similar to my previous comment, fan only doesn't work for me here - easy fix though. You've done great with the heater integration in general though thanks!